frontend/src/app/person/person.component.html

```diff
@@ -72,12 +101,21 @@ <h2>Contact</h2>
       <div>{{ person.phoneNumber }}</div>
     </div>
 
-    <ng-container *ngIf="person.mediationEnabled">
-      <h2>Logement</h2>
+    <div class="d-flex mb-2" id="accompanying"  *ngIf="person.mediationEnabled">
+      <div class="mr-3"><span class="fa fa-users" title="accompagnant"></span></div>
+      <div>{{ person.accompanying }} (accompagnant)</div>
+    </div>
+
+    <hr class="separator">
+
+    <h2>Logement</h2>
       <p id="housing">
         {{ person.housing | displayHousing }}
         <ng-container *ngIf="person.housingSpace"> &mdash; {{ person.housingSpace }} m<sup>2</sup></ng-container>
       </p>
+      <ng-container *ngIf="person.hostName">Hébergé par {{ person.hostName }}</ng-container>
```

Comment:
Hébergé(e)? Or, if you're really brave, you could use the gender to decide what to display (Hébergé or Hébergée)

Reply:
for te moment, I just use Hébergé(e)